ImageBuilderRAM/ShareImages-RAM.yml

```diff
@@ -0,0 +1,149 @@
+AWSTemplateFormatVersion: 2010-09-09
+Parameters:
+  OrgId:
+    Description: Id of your Organization in the following format o-xxxxxxx
+    Type: String
+    AllowedPattern: ^o-[a-z0-9]{10,32}$
+    ConstraintDescription: organization id should start with "o-" followed by 10-32 alphanumeric characters
+    Default: o-abcde12345
+
+  ManagementAccountId:
+    Description: Management Account ID
+    Type: String
+    AllowedPattern: ^\d{12}$
+    ConstraintDescription: 12 Digit Account number of the Management Account
+    Default: 123456789100
+  
+  ResourceShareName:
+    Type: String
+    Default: ImageShare
+
+  SNSTopicName:
+    Description: ImageBuilder SNS Topic Name
+    Type: String
+    Default: ImageBuilderNotifications
+
+Resources:
+  ImagesResourceShare:
+    Type: AWS::RAM::ResourceShare
+    Properties:
+      Name: !Ref ResourceShareName
+      AllowExternalPrincipals: False
+      PermissionArns:
+        - arn:aws:ram::aws:permission/AWSRAMDefaultPermissionImageBuilderImage
```

Comment:
Instead of hardcoding the `aws` partition into the arn, use the `arn:${AWS::Partition}` pseudo parameter which allows this sample to work in other partitions such as China or AWS GovCloud (US).

For more information on pseudo parameters, see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/pseudo-parameter-reference.html

Reply:
All Requested modifications have been completed